src/synth.ts

```diff
@@ -407,6 +425,18 @@ function initializeAperiodic(audioContext: BaseAudioContext) {
 
     return new AperiodicWave(audioContext, freqs, amps, maxNumberOfVoices, tolerance)
   })
+
+  getInharmonicSimpleWaveformNames().forEach((id) => {
+    APERIODIC_WAVES[id] = computed(() => {
+      const freqs = getInharmonicSimple(id)
```

Comment:
`const spectrum = ...` instead. These are not frequencies. They are frequency ratios.

Reply:
I just copied this part from `piano`, I’ll edit it there too then?

Reply:
Sure. Please do. I must've been tired when I wrote that piece of code.